include/flang/Lower/PFTBuilder.h

```diff
@@ -168,102 +125,114 @@ struct Evaluation {
                       },
                       u);
   }
-  
   template <typename A>
   constexpr const A *getIf() const {
-    if constexpr (!std::is_same_v<A, CGJump>) {
-      if (auto *ptr{std::get_if<MakeRefType<A>>(&u)})
-        return *ptr;
-    } else {
-      return std::get_if<CGJump>(&u);
+    if (auto *ptr{std::get_if<MakeRefType<A>>(&u)}) {
+      return *ptr;
```

Comment:
Is there a way to simplify this? It seems like it'd be a common idiom to return a variant member if and only if it has some type.

Reply:
Jean again, most likely.  If someone has a concrete suggestion, I'll make the change.

Reply:
Right. I was thinking the ternary return might be hidden behind some call. (Personally, I prefer the ternary, but I thought <pick a standard> might be forbidding it.)

Reply:
That looks good; thanks!